adhocracy-plus/assets/scss/_form.scss

```diff
@@ -4,7 +4,6 @@ label {
 }
 
 %input-base {
-    margin: 0 0 $spacer;
```

Comment:
this will affect all inputs, do we want to do that? did you check different forms to make sure it doesn't break the styling everywhere?

Reply:
i double checked it in our components library. And it seems that it does not have any real effect on other inputs.

There is a class overwriting this to set it to 0 anyways. Except for type radio or checkbox. But those are not visible anyways due to own styling.

<img width="311" alt="Bildschirmfoto 2022-12-08 um 16 41 06" src="https://user-images.githubusercontent.com/52801593/206490424-1f251c87-b57a-4f83-969d-701d42517657.png">